specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/examples/PatchAProximityPlacementGroup.json

```diff
@@ -0,0 +1,27 @@
+{
+   "parameters":{
+      "subscriptionId":"{subscription-id}",
+      "resourceGroupName":"myResourceGroup",
+      "api-version":"2018-10-01",
+      "proximityPlacementGroupName":"myProximityPlacementGroup",
+      "parameters":{
+         "location":"westus",
```

Comment:
Remove location, since `ProximityPlacementGroupUpdate`  (patch body) does not have one.

Reply:
Fixed